### DISCUSSION TITLE
希望视频播放器采用dplayer播放器

### DISCUSSION THREAD

**Discussion by liangfyx** [spencerwooo/onedrive-vercel-index#263] [Ideas]
dplayer播放器优点如下
1.dplayer播放器是一个比较成熟的网页端播放器，其支持多种视频格式，且开源免费
2.同时其支持的功能很多，比如：弹幕，截屏，快捷键，画质切换，缩略图和字幕等
[开原地址](https://github.com/DIYgod/DPlayer)

**Comment by spencerwooo** [comment]
dplayer 两年没维护了兄弟，不过确实考虑更换视频播放器，可能用 https://github.com/video-react/video-react

**Reply by liangfyx** [reply]
期待一下下~

**Comment by spencerwooo** [comment]
DPlayer 真香了😂

https://github.com/spencerwooo/onedrive-vercel-index/discussions/325

**Reply by lyizhuo** [reply]
Dplayer好像无法播放H265编码的视频？显示Video load failed，h264的视频能正常播放
<img width="955" alt="image" src="https://user-images.githubusercontent.com/64514694/152362202-a0ad8b48-381a-4117-837c-d29c744d6d4a.png">

**Reply by spencerwooo** [reply]
之前的项目里 H265 都不能解码，不是 DPlayer 的问题

**Reply by spencerwooo** [reply]
H.265 will only work in Safari and IE - https://caniuse.com/?search=hevc

<img width="1506" alt="CleanShot 2022-02-03 at 22 32 05@2x" src="https://user-images.githubusercontent.com/32114380/152362860-b69ec0e8-6ff6-4754-b51b-9e4ebf82dc23.png">

**Reply by lyizhuo** [reply]
好像有在其他浏览器解码h265视频的方案，后续能实现嘛😁
https://github.com/goldvideo/h265player

**Reply by liangfyx** [reply]
wow，支持dplayer了吗，立马去更新，手动狗头
